specs/lightclient/sync-protocol.md

```diff
@@ -97,6 +99,15 @@ class LightClientStore(Container):
     valid_updates: List[LightClientUpdate, MAX_VALID_LIGHT_CLIENT_UPDATES]
 ```
 
+## Helper functions
+
+### `get_subtree_index`
+
+```python
+def get_subtree_index(generalized_index: GeneralizedIndex) -> uint64:
+    return uint64(generalized_index % 2**(floorlog2(generalized_index)))
```

Comment:
Why is this not also a "sundry" function? Surely it's part of the "generalized index" suite.